src/pal/src/file/directory.cpp

```diff
@@ -369,6 +341,35 @@ GetCurrentDirectoryA(
     return dwDirLen;
```

Comment:
dwDirLen is no longer set. Need to restore `dwDirLen = strlen( current_dir );` above.